driver_cpl/driver/seq_diag_mct.F90

```diff
@@ -880,6 +883,9 @@ subroutine seq_diag_lnd_mct( lnd, frac_l, infodata, do_l2x, do_x2l)
                                                                     - dl*l2x_l%rAttr(index_l2x_Flrl_rofgwl,n) &
                                                                     - dl*l2x_l%rAttr(index_l2x_Flrl_rofsub,n) &
                                                                     - dl*l2x_l%rAttr(index_l2x_Flrl_rofdto,n)
+         if (index_l2x_Flrl_irrig /= 0) then
+            if = f_wroff ; budg_dataL(if,ic,ip) = budg_dataL(if,ic,ip) - dl*l2x_l%rAttr(index_l2x_Flrl_irrig,n)
```

Comment:
I really don't like the use of a variable 'if'   It took me a while to figure this code out - I know it's not your doing but can we change that?